packages/text-area/src/TextArea.js

```diff
@@ -25,15 +25,32 @@ export default class TextArea extends Component {
     /**
      * The visual variant of the textarea
      */
-    variant: PropTypes.oneOf(variantTypes)
+    variant: PropTypes.oneOf(variantTypes),
+    /**
+     * Adds custom/overriding styles
+     */
+    stylesheet: PropTypes.func
   };
 
   static defaultProps = {
     variant: "line"
   };
 
+  constructor(props) {
+    super(props);
+    this.textareaStylesheet = this.textareaStylesheet.bind(this);
+  }
+
+  textareaStylesheet(styles, props, themeData, theme) {
+    const { stylesheet } = this.props;
```

Comment:
Since this just the logic for choosing between which `stylesheet` functions to use, I am wondering if it would make sense to just put the `const` and logic in the `render` method. That way we would need the `constructor`.

Reply:
Sure, i've updated for the changes.